srcpkgs/spago/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'spago'
+pkgname=spago
+version=0.16.0
+revision=1
+hostmakedepends="
+	curl
+	ncurses-libtinfo-libs
```

Comment:
This shouldn't be necessary with the patch you apply. Also, this would be the wrong package and the wrong place to declare it.